### PR TITLE
Feature/reset password page

### PR DESCRIPTION
**Pull request by rahulkurumkar**
Implemented the reset password functionality
TODO:
Need to set reset password URL with unique id generated after clicking on reset password link in the email.
ResetPassword

### CODE REVIEW

**Comment by xantorres** [comment]
Looks good, except for the styling on the error shown after submitting, which look a bit off.

File: src/app/plugins/openstack/components/ResetPasswordPage.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    updateParams({
+      loading: true
+    })
+
+    const body = { username: params.emailId, password: params.confirmPassword }
+
+    try {
+      const response = await axios.post(resetPasswordApiUrl, body)
+
+      if (response.status === 200) {
```
[the remark this answers is not in the document]
Comment (rahulkurumkar) [reply src/app/plugins/openstack/components/ResetPasswordPage.js]: Incorporated the review changes

File: src/app/plugins/openstack/components/ResetPasswordPage.js [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+      <Grid container justify="center" className={classes.root}>
+        <Grid item md={5} lg={4}>
+          <Paper className={classes.paper}>
+            <img src="/ui/images/logo-color.png" className={classes.img} />
+            <Typography variant="subtitle1" align="center">
+                Password Reset
+            </Typography>
+            <ValidatedForm onSubmit={handleFormSubmit}>
+              {({ values }) => <>
+                {!params.isSetNewPasswordSuccessful ? (<><TextField
```
[the remark this answers is not in the document]
Comment (rahulkurumkar) [reply src/app/plugins/openstack/components/ResetPasswordPage.js]: Formatted code properly

**Comment by rahulkurumkar** [comment]
Updated UI GIF
ResetPassword1

**Pull request merged**